### PR TITLE
Update features table

### PR DESCRIPTION
**Pull request by asmeurer**
Now it has an extensive listing of all the major features. I've also alphabetized them (look at the individual commits for a more readable diff).

### CODE REVIEW

**Comment by asmeurer** [comment]
In https://github.com/sympy/sympy-paper/pull/116 I've made the table text wider. I will merge that into this PR as soon as It's merged into master.

File: features.tex [hunk 1]
```diff
@@ -1,67 +1,117 @@
...
+and
+other properties. May also compute Gosper's normal form~\cite{petkovvsek1996bak} for two univariate polynomials.\\
+
+Cryptography & Represents block and stream ciphers, including shift, Affine,
+substitution, Vigenere's, Hill's, bifid, RSA, Kid RSA,
+linear-feedback shift registers, and Elgamal encryption\\
+
+Differential Geometry & Classes to represent manifolds, metrics, tensor
+products, and coordinate systems.\\
+% TODO: Someone verify that this is a good summary of the diffgeom module
```
Comment (asmeurer) [inline features.tex:43]: Can someone verify that this is a good description here?

**Comment by asmeurer** [comment]
I feel like there also needs to be a more consistent style in the way text in the table is written.

**Comment by scopatz** [comment]
This LGTM

**Comment by asmeurer** [comment]
What is going on with Travis? 

```
Hit http://ppa.launchpad.net trusty/main amd64 Packages
Hit http://ppa.launchpad.net trusty/main Translation-en
Hit http://ppa.launchpad.net trusty/main amd64 Packages
Hit http://ppa.launchpad.net trusty/main Translation-en
Ign http://dl.google.com stable/main Translation-en_US
Hit http://ppa.launchpad.net trusty/main amd64 Packages
Ign http://dl.google.com stable/main Translation-en
Hit http://ppa.launchpad.net trusty/main Translation-en
Fetched 1,334 B in 2s (565 B/s)
W: There is no public key available for the following key IDs:
1397BC53640DB551
W: Failed to fetch http://dl.google.com/linux/chrome/deb/dists/stable/main/binary-amd64/Packages  Hash Sum mismatch
E: Some index files failed to download. They have been ignored, or old ones used instead.
The command "sudo apt-get update && sudo apt-get install -y --no-install-recommends texlive-fonts-recommended texlive-latex-extra texlive-fonts-extra texlive-humanities texlive-science latex-xcolor dvipng texlive-latex-recommended python-pygments" failed and exited with 100 during .
Your build has been stopped.
```

**Comment by scopatz** [comment]
No idea.  Travis has been sort of slow all day for me.

**Comment by asmeurer** [comment]
Reset Travis and it built. Merging.

**Pull request merged**